### ISSUE TITLE
`screenfetch` installed from nixpkgs gives error message

### ISSUE THREAD

**Issue by btlvr** [NixOS/nixpkgs#76561]
**Describe the bug**
I've installed the `screenfetch` package from nixpkgs. When I run `screenfetch`, the following is printed before the expected screenfetch output:

```
awk: cmd. line:7: warning: regexp escape sequence `\"' is not a known regexp operator
```

**To Reproduce**
1. install `screenfetch`
2. run `screenfetch`

**Expected behavior**
There should not be an error message. 

**Metadata**
```
 - system: `"x86_64-linux"`
 - host os: `Linux 4.19.88, NixOS, 19.09.1548.3a1861fcabc (Loris)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.3`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos
```

**Comment by Ma27** [comment]
Those issues seem to be fixed in screenfetch-3.9.1 (I can reproduce the issue on 19.09, but not on master). Will backport the update, that should fix the issue within the next channel update of nixos-19.09 :)

**Comment by Ma27** [comment]
With 2314fb1bbfdac43321cac479bce9a1f1d2b3a878 being backported, the issue isn't reproducible anymore for me.
